core/templates/vector.h

```diff
@@ -99,7 +99,7 @@ class Vector {
 	Error resize(Size p_size) { return _cowdata.resize(p_size); }
 	Error resize_zeroed(Size p_size) { return _cowdata.template resize<true>(p_size); }
 	_FORCE_INLINE_ const T &operator[](Size p_index) const { return _cowdata.get(p_index); }
-	Error insert(Size p_pos, T p_val) { return _cowdata.insert(p_pos, p_val); }
+	Error insert(Size p_pos, const T &p_val) { return _cowdata.insert(p_pos, p_val); }
```

Comment:
I see now that this change would cause a regression of #34618. Using an explicit copy isn't a great solution, but this should be for a follow-up PR. Therefore, I will remove the change for now.